amqp/src/main/scala/akka/stream/alpakka/amqp/impl/AbstractAmqpAsyncFlowStageLogic.scala

```diff
@@ -30,6 +30,9 @@ import scala.concurrent.duration.FiniteDuration
  */
 @InternalApi private object AbstractAmqpAsyncFlowStageLogic {
   type DeliveryTag = Long
+  val DefaultBufferSize: Int = 10
+  val DefaultConfirmationTimeout: FiniteDuration = 100.millis
```

Comment:
The defaults should go in `AmqpConnectorSettings` so they are always defined instead.

Reply:
I wanted to make the values in `AmqpWriteSettings` optional to indicate they are sometimes ignored, but indeed, when I moved them to settings it looks cleaner.